Makefile

```diff
@@ -240,6 +240,7 @@ image-vmware:
 		--privileged $(DOCKER_ARGS) \
 		autonomy/installer:$(TAG) \
 		ova
+	@rm -rf $(PWD)/build/talos.raw
```

Comment:
Here or in entrypoint?

Reply:
We do this same thing for GCP in the Makefile. So for the sake of consistency here. I started reworking the entrypoint which should clean up our makefile a bit, but this PR gets us able to release at least.